src/main/java/core/basesyntax/operations/Operation.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface Operation {
+    void executeOperation(FruitTransaction fruitTransaction);
+}
```

Comment:
Better to rename this interface. Try somthing like TransactionHandle or TransactionExecuter, cause you work with transaction here. The same with all implementation